src/main/kotlin/com/openlattice/assembler/Assembler.kt

```diff
@@ -325,15 +325,15 @@ class Assembler(
         }
 
         val sql = selectEntitySetWithCurrentVersionOfPropertyTypes(
-                entitySetIds.map { it to Optional.empty<Set<UUID>>() }.toMap(),
+                entitySetIds.associate { it to Optional.empty<Set<UUID>>() },
                 propertyFqns,
                 authorizedPropertyTypes.values.map(PropertyType::getId),
                 mapOf(entitySetId to authorizedPropertyTypes.keys),
                 mapOf(),
                 EnumSet.allOf(MetadataOption::class.java),
                 authorizedPropertyTypes.mapValues { it.value.datatype == EdmPrimitiveTypeKind.Binary },
                 entitySet.isLinking,
-                false //Always provide entity set id
+                entitySet.isLinking //false //Always provide entity set id
```

Comment:
I think this will break materialize in AssemblerConnectionManager, because we select entity_set_id and linking_id for materialized view and with this, omitEntitySetId will always be true for linking entitysets.

Reply:
Yup. You'll have to modify how it's done. @katherinebernstein made a change that does a group by during linking entity sets to fix indexing, which she only tested with omitEntitySetId = true for linked entity sets.